docs/reference/esql/functions/kibana/definition/qstr.json

```diff
@@ -0,0 +1,35 @@
+{
+  "comment" : "This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.",
+  "type" : "eval",
+  "name" : "qstr",
+  "description" : "Performs a query string query. Returns true if the provided query string matches the row.",
+  "signatures" : [
+    {
+      "params" : [
+        {
+          "name" : "query",
+          "type" : "keyword",
+          "optional" : false,
+          "description" : "Query string in Lucene query string format."
+        }
+      ],
+      "variadic" : false,
+      "returnType" : "boolean"
+    },
+    {
+      "params" : [
+        {
+          "name" : "query",
+          "type" : "text",
+          "optional" : false,
+          "description" : "Query string in Lucene query string format."
+        }
+      ],
+      "variadic" : false,
+      "returnType" : "boolean"
+    }
+  ],
+  "examples" : [
+    "from books \n| where qstr(\"author: Faulkner\")\n| keep book_no, author \n| sort book_no \n| limit 5;"
+  ]
+}
```

Comment:
Somehow the `"preview" : true` is not here, I'll update here if I find out why.

Reply:
This was due to the docs being generated without the merged version from main - fixed in 28b72f9e945ca27a824c6bb602a5e6e0e9f47bdf